data/instrumentation.yaml

```diff
@@ -62,6 +62,6 @@ languages:
   swift:
     name: Swift
     status:
-      traces: beta
-      metrics: alpha
-      logs: not yet implemented
+      traces: stable
+      metrics: outdated
```

Comment:
hmmm, how should I interpret this as an end-user? Is it something I can count on getting up to date soon?

Reply:
for .NET we have a „mixed *“ and then a description what that means, we could do the same here, put a * besides the outdated and a comment on the index page what this means. Not perfect tbh but it’s temporary…

Otherwise you could keep it as „in development“. This was the situation for JS for some time, they also had done some major reworks.

Reply:
Yeah, I would say that if there's nothing special to say about it here, then "in development" would be my preference. @bryce-b any thoughts?

Reply:
Sure

Reply:
"outdated" is confusing. Please use the project's stability guarantee terminology - see https://opentelemetry.io/docs/reference/specification/versioning-and-stability/

Reply:
> "outdated" is confusing. Please use the project's stability guarantee terminology - see https://opentelemetry.io/docs/reference/specification/versioning-and-stability/

So instead of "in development" to "experimental"? We have used "in development" before as well: for me it has the nuance of "we are working on that, but there's not a lot you can _experiment_ with yet", so it's somewhere between "not available" and "experimental"

Reply:
then let's go for experimental